react/index.js

```diff
@@ -2,6 +2,10 @@ export { default as bell } from './library/bell';
 export { default as checkCircle } from './library/check-circle';
 export { default as check } from './library/check';
 export { default as comment } from './library/comment';
+export { default as doubleChevronDown } from './library/double-chevron-down';
```

Comment:
I would suggest renaming this to `chevron{angle}Double` instead. This will help users working with (IDE) autocomplete to know that there is a double chevron variation that they can use as well.

Reply:
Great, should I rename the files too?

Reply:
Absolutly.